### PR TITLE
Render typesv2

### PR DESCRIPTION
**Pull request by MilesAger**

### CODE REVIEW

File: client/src/PokeTypes.jsx [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
 
-  const [isVisible, setIsVisible] = useState(false);
+  const [types, setTypes] = useState([]);
+
+  useEffect(() => {
+
+    const getAllTypes = async () => {
+      try {
+        const allTypes = await axios.get('/api/types');
+        setTypes(response.data);
```
Comment (copelka) [inline client/src/PokeTypes.jsx]: where is response coming from?

Comment (MilesAger) [reply client/src/PokeTypes.jsx]: haven't set it up yet, but I am able to verify the Type collection is in the database.

File: server/db/dbHelperFuncs.js [hunk 2]
```diff
@@ -99,6 +99,9 @@ const getUsersMsg = (id, cb) => {
 };
 
 
+const addType = () => {
+
```
Comment (colinwilliams91) [inline server/db/dbHelperFuncs.js]: what's this function going to do?

**Review by Musique243 — APPROVED** [review]

File: server/db/dbHelperFuncs.js (hunk 2 above)
Comment (MilesAger) [reply server/db/dbHelperFuncs.js]: removed it.

**Review by sniehus — APPROVED** [review]

**Pull request merged**